rails_application/app/read_models/vat_rates/configuration.rb

```diff
@@ -0,0 +1,15 @@
+module VatRates
+  class AvailableVatRate < ApplicationRecord
+    self.table_name = "available_vat_rates"
+
+    def to_value
```

Comment:
It is not used anymore, you can remove this method in next PR :)

Reply:
I already removed it :)